### PR TITLE
fix typo. reifiee => reifier

### PR DESCRIPTION
**Pull request by zuvizudar**
Sorry if it's wrong.

### CODE REVIEW

**Comment by zuvizudar** [comment]
I did sign the CLA.

**Comment by SethTisue** [comment]
/rebuild

```
!! 1491 - run/t7185.scala                           [output differs]
```

File: src/compiler/scala/reflect/reify/phases/Calculate.scala [hunk 1]
```diff
@@ -26,7 +26,7 @@ trait Calculate {
     }
 
   /**
-   *  Merely traverses the reifiee and records symbols local to the reifee along with their metalevels.
+   *  Merely traverses the reifier and records symbols local to the reifee along with their metalevels.
```
Comment (xeno-by) [inline src/compiler/scala/reflect/reify/phases/Calculate.scala]: I did mean the "reifee" here as in "something that is being reified".

Clearly, the term is cryptic enough to confuse readers (it's most likely not even an English word), so I'd suggest replacing it with "the tree that is being reified"

**Review by xeno-by — CHANGES_REQUESTED** [review]

File: src/compiler/scala/reflect/reify/phases/Calculate.scala (hunk 1 above)
Comment (SethTisue) [reply src/compiler/scala/reflect/reify/phases/Calculate.scala]: perhaps "target"

Comment (zuvizudar) [reply src/compiler/scala/reflect/reify/phases/Calculate.scala]: Do you mean you'd like me to change "reifier" to "reifee"  for now ?
I'm not sure I understand it completely, but I can only say that "reifiee" is typo.

Comment (zuvizudar) [reply src/compiler/scala/reflect/reify/phases/Calculate.scala]: sorry..,I can not understand.
I am waiting for your instructions.

Comment (SethTisue) [reply src/compiler/scala/reflect/reify/phases/Calculate.scala]: I'm suggesting "Merely traverses the target", Eugene suggested "Merely traverses the tree being reified". Either suggestion seems acceptable to me

**Review by SethTisue — APPROVED** [review]
we can merge once Jenkins passes.

**Comment by zuvizudar** [comment]
What should I do to make Jenkins pass ?

**Comment by SethTisue** [comment]
Jenkins is down right now; you can monitor https://contributors.scala-lang.org/t/jenkins-is-down/1551 for updates

**Comment by SethTisue** [comment]
thanks @zuvizudar!

**Pull request merged**